components/match2/commons/match_group.lua

```diff
@@ -94,19 +94,18 @@ function MatchGroup.MatchPage(args)
 	end
 
 	local bracketId, matchId = getBracketIdFromPage()
+	bracketId = args.bracketid or bracketId
+	matchId = args.matchid or matchId
+	local fullMatchId = bracketId .. '_' .. matchId
 
-	local options = {storeMatch1 = false}
-	local matches = MatchGroupInput.readMatchpage(args.bracketid or bracketId, args.matchid or matchId, args)
+	local options = {storeMatch1 = false,  storeMatch2 = true, storePageVar = true, bracketId = bracketId}
```

Comment:
should we potentially add an option to disable `storeMatch2`?

Reply:
I don't see the use of it, as the data is needed to populate the real bracket/matchlist with data